starter-code/functions-and-arrays.js

```diff
@@ -63,7 +84,14 @@ console.log(averageLength);
 
 // Unique Arrays
 function uniquifyArray (array) {
-
+  var uniqueArray = [];
+  array.forEach(function(element) {
+    var uniqueElement;
+    var index = element.indexOf();
```

Comment:
It seems you got stuck on this. You need to use `indexOf` on the array, not the element.

Reply:
I found the solution in the evening)))
https://repl.it/Kh8w/4

Reply:
Awesome! Please add those changes to your repo by doing a new commit and pushing. This will automatically update the PR. Once you're done I will validate your work ;)